### DISCUSSION TITLE
Weapon proposal: Double-barrel Shotgun/Sawed-off

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1155] [Ideas and Feature Requests]
I've been thinking lately, since we have a standart shotgun, 8 shells, small spread, focused fire can kill a target even at a distance with 2 or 3 shots, why not bring it's other counterpart, the Double barrel shotgun?
Strong at personal/close range, but almost useless at a distance. 
It has 2 shells, plenty of pellets per shot (30 ~~ ), but a MASSIVE spread to compensate.

This is the prototype variables i came up with for the proposal:
`shotgunammoclip 2
shotguncolour 0xF0A000
shotgundamage1 100
shotgundelayattack1 400
shotgundelayreload 1500
shotgunpartcol1 0xF0A000
shotgunpartcol2 0xF0A000
shotgunrays1 30
shotgunspread1 70.0
`
Also made a very rough mesh of the thing.

![20181218175542](https://user-images.githubusercontent.com/24359960/50179538-6e72cc80-02ee-11e9-8b75-02468da71ea6.png)

 And here is the final result (audio got weird tho) :  https://youtu.be/u3ijxrAlIao
Files for those that want to play around with it:
[shotgun-double-barrel.zip](https://github.com/red-eclipse/world/files/2692078/shotgun-double-barrel.zip)
Place it on my games/Red Eclipse/weapons for it to work. Or just replace the actual file, if this doesn't work.

So, i wonder if this would be worth to work into the 2.0 release?

**Comment by lambda-11235** [comment]
I don't think it really works that well. From play testing I can tell you that players will simply move back to get out of the range of the shotgun, and they don't even have to go that far, even 2m of distance is enough for the range to drop off. In really cramped situations I keep hitting my allies because the spread is so high.

tldr: The shotgun is really hard to control.

**Comment by SniperGoth** [comment]
Yup, it's just a proof of concept at this stage, so feedback is welcome.
Maybe the spread value is too high, i guess.
